CHANGELOG.md

```diff
@@ -128,6 +128,14 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Style any custom HTML in your cookie banner
+
+We have removed the default font styles from the cookie banner Sass, to make it more obvious when classes/styles haven't been added to any custom HTML.
+
+If you are passing custom HTML into the cookie banner component (for example, using the `html` Nunjucks param), you must make sure you are applying the appropriate classes/styles to that HTML, such as adding the `govuk-body` class to any paragraph tags. You must do this to prevent your cookie banner displaying with unstyled text.
```

Comment:
```suggestion
##### If you're passing custom HTML (for example, using the `html` Nunjucks parameter) into the cookie banner component

Make sure you're applying the appropriate classes and styles to that HTML. For example, add the `govuk-body` class to any paragraph tags. You must do this to prevent your cookie banner displaying with unstyled text.
```
Just suggesting this to break up the sentences a little. Also took out the first "You must", in case it suffices to link the remaining "You must" with what happens if you users do not do this.

Reply:
Is there any other way users could pass custom HTML without using the `html` param?

Reply:
@EoinShaughnessy If the user was using the HTML version of the component (not the Nunjucks macros) they can edit the HTML however they want and put whatever they want in there. Hopefully these users would already be more aware of styling their HTML, but it's possible someone could have added some custom HTML which previously would have looked "correct" but following this update won't look styled any more. They'd need to take the same actions as anyone using the Nunjucks macro though.

Reply:
I'm not 100% sure I see the benefit of splitting this out into its own heading? At least to me, it kind of suggests that there might be another heading later on with different actions if you do something else, which isn't the case?

Reply:
That's a good point - yeah, best not to split out then.